Specs/Number/English/NumberModel.json

```diff
@@ -1658,4 +1658,31 @@
     }
   ]
 }
+,
+{
```

Comment:
Please add cases for when the "of" will not be tagged as part of the entity if the new mode if not enabled.